terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -124,14 +124,9 @@ locals {
       "output_file_prefix": null,
       "output_directory": null,
       "fastq_list_rows": null,
-      "sites_somalier": {
-        "class": "File",
-        "location": "gds://development/reference-data/somalier/sites.hg38.rna.vcf.gz"
-      },
-      "reference_fasta": {
-        "class": "File",
-        "location": "gds://development/reference-data/genomes/hg38/hg38.fa"
-      },
+      "enable_duplicate_marking": true,
+      "enable_map_align_output": true,
+      "enable_sort": true,
```

Comment:
Right. So I understood that, these flags were there previously and, now make them explicit in workflow input template config to better capture these, correct?

Reply:
Yep that's correct